### PR TITLE
Docs for AVS Dashboard Onboarding

### PR DESCRIPTION
**Pull request by wesfloyd**

### CODE REVIEW

**Review by dabit3 — APPROVED** [review]

File: docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "logo": "https://holesky-operator-metadata.s3.amazonaws.com/eigenlayer.png",
+    "twitter": "https://twitter.com/eigenlayer"
+}
+```
+
+## Quorums
+The design for a V2 marketplace that supports more expressive views of stake (quorums, insurance, payments, etc) has not been finalized. Recommendation to AVS teams is to upgrade their contracts for V2 marketplace to meet interface requirements for indexing.
+
+## Frontend Onboarding - Whitelist
+We currently have a whitelist on the testnet version of the frontend, so once you have done the above, reach out to Nima and provide your service manager address to be added to the whitelist.
```
Comment (NimaVaziri) [inline docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md:56]:
```suggestion
## Holesky Dashboard onboarding
Once you've gone through the above steps, fill out [this form](https://forms.gle/8BJSntA3eYUnZZgs8) so the metadata you submitted for your AVS can be reflected on the Holesky dashboard
```

File: docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 6
+title: AVS Dashboard Onboarding
+---
+
+
+This document defines interfaces that AVSs must implement to index their data for the V1 AVS Marketplace
```
Comment (NimaVaziri) [inline docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md:7]:
```suggestion
This document defines interfaces that AVSs must implement to index their data needed for [dashboard](https://holesky.eigenlayer.xyz/avs) onboarding
```

File: docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 6
+title: AVS Dashboard Onboarding
```
Comment (NimaVaziri) [inline docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md:3]:
```suggestion
title: AVS Holesky Dashboard Onboarding
```

**Review by joan-el — DISMISSED** [review]

**Comment by wesfloyd** [comment]
Hi @NimaVaziri any other changes we should make for this one? Thx

**Review by NimaVaziri — APPROVED** [review]

**Pull request merged**